### PR TITLE
Development

### PR DESCRIPTION
**Pull request by AndyTheNerd**
Usage added to commands
New random and easter egg commands

### CODE REVIEW

File: commands/8ball.js [hunk 1]
```diff
@@ -3,8 +3,9 @@ const client = new Discord.Client();
 
 module.exports = {
     name: '8ball',
-    description: 'Leave the most important decisions up to random! To run, type `!8BALL`',
+    description: 'Leave the most important decisions up to random!',
     cooldown: 5,
+    usage: ` should I invite this bot to my server?`,
```
Comment (Patross) [inline commands/8ball.js:8]: `!8ball should I invite this bot to my server?`

**Comment by Patross** [comment]
Help command errors out when the users types !help [alias] rather than the main command name

File: commands/about.js [hunk 2]
```diff
@@ -3,7 +3,8 @@ const helpers = require('../helpers/helpers');
 const client = new Discord.Client();
 module.exports = {
 	name: 'about',
-	description: 'Storehaus is a bot that queries various APIs to enhance your discord server. This bot was created to help gamers discover, share, and enjoy info on their favorite games! To run, type `!about',
+	description: 'Storehaus is a bot that queries various APIs to enhance your discord server. This bot was created to help gamers discover, share, and enjoy info on their favorite games!',
+	usage: ` `,
```
Comment (Patross) [inline commands/about.js:7]: blank usage

**Comment by Patross** [comment]
for !age2 when typing `age2 all civs`
image
for !age2 when typing `age2 japanese`
image

**Review by Patross — CHANGES_REQUESTED** [review]
Refer to comments for changes.

**Comment by AndyTheNerd** [comment]
@Patross That's because you have to do !help [command name]. There's really no work around that I know of.

As for Age2, it's API is down.

**Pull request merged**